driver/compenv.ml

```diff
@@ -350,6 +350,15 @@ let read_one_param ppf position name v =
       | Some setting -> color := Some setting
       end
 
+  | "error-style" ->
+      begin match parse_error_style v with
+      | None ->
+          Printf.ksprintf (print_error ppf)
+            "bad value %s for \"error-style\", \
+             (expected \"contextual\" or \"classic\")" v
```

Comment:
(Here one could reuse `Clflags.error_style_reader.usage` if the reader type was exposed. Not sure that is important.)